TelCo.ColorCoder/NewFeature.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Globalization;
+namespace Telco.ColorCoder
+{
+    class ColorMapManual
+    {
+        public static string getFormattedLine(int pairNumber, string color_pair)
+        {
+            CultureInfo provider = new CultureInfo("en-us");
+            string temp = pairNumber.ToString(provider) + " - " + color_pair + "\n"; 
+            return temp;
+        }
+        public static string GetManual(int numberOfMajorColors, int numberOfMinorColors)

```

Comment:
The attempt here is to replicate the code in a FakeGetManual and test it. Good attempt but can be better 👍 
The fake and the real code can start diverging in future.
Try a different approach: Separate the formation of the color codes and pair numbers from the formatting / console output.
Then you can pass the mapping of pair numbers to colors to the console-output function and test it with lesser data.
Let me know when something is not clear.